__tests__/commands/dev-env-sync-sql.ts

```diff
@@ -1,74 +1,24 @@
 import { replace } from '@automattic/vip-search-replace';
-import fs, { ReadStream } from 'fs';
+import fs from 'fs';
 import Lando from 'lando';
-import { WriteStream } from 'node:fs';
-import { Interface } from 'node:readline';
-import { PassThrough } from 'stream';
+import path from 'path';
 
 import { DevEnvImportSQLCommand } from '../../src/commands/dev-env-import-sql';
 import { DevEnvSyncSQLCommand } from '../../src/commands/dev-env-sync-sql';
 import { ExportSQLCommand } from '../../src/commands/export-sql';
-import { unzipFile } from '../../src/lib/client-file-uploader';
-import { getReadInterface } from '../../src/lib/validations/line-by-line';
-
-/**
- *
- * @param {Array<{name, data}>} eventArgs Event arguments
- * @param {timeout}             timeout
- *
- * @return {Stream} A passthrough stream
- */
-function getMockStream( eventArgs: { name: string; data?: string }[], timeout = 10 ) {
```

Comment:
I had to refactor the tests by quite a bit - it's mocking too much and adjusting the mock is starting to take too much time. Instead, I don't mock it but provide a small fixture that is also speedy.